### ISSUE TITLE
Crash on new message receive

### ISSUE THREAD

**Issue by maxmaxandr** [GetStream/stream-chat-android#490]
I've got a crash when receiving a new message from another user. 

com.github.getstream:stream-chat-android:4.2.7
com.github.GetStream:stream-chat-android-livedata:0.4.5

```
I/Chat:Events: onMessage: {"type":"notification.message_new","cid":"messaging:256","message":{"id":"bfb218b9-b280-4398-ad36-ddd46bde026a","text":"123234" .. 

I/Chat:ChatDomain EventHandler: Handling event of type notification.message_new for cid messaging:256, event batch size is 1
E/AndroidRuntime: FATAL EXCEPTION: DefaultDispatcher-worker-2
    java.security.InvalidParameterException: message.cid cant be empty
        at io.getstream.chat.android.livedata.repository.MessageRepository.insert(MessageRepository.kt:69)
        at io.getstream.chat.android.livedata.EventHandlerImpl.updateOfflineStorageFromEvents$livedata_release(EventHandlerImpl.kt:201)
        at io.getstream.chat.android.livedata.EventHandlerImpl$updateOfflineStorageFromEvents$1.invokeSuspend(Unknown Source:12)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:56)
        at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:738)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)

```

**Comment by elevenetc** [comment]
Thanks for report.

**Comment by tschellenbach** [comment]
@tbarbugli

**Comment by tschellenbach** [comment]
@maxmaxandr what's the full payload for the notification.message_new event?

**Comment by tschellenbach** [comment]
Fixed in 0.4.8: https://github.com/GetStream/stream-chat-android-livedata/releases/tag/0.4.8
